python_utils_test.py

```diff
@@ -256,6 +256,15 @@ def func(a, b, *c, **d): # pylint: disable=unused-argument
         self.assertEqual(
             python_utils.get_args_of_function(func), ['a', 'b'])
 
+    def test_create_enum_method(self):
+        """Test create_enum method."""
+        enums = python_utils.create_enum('first', 'second', 'third')
+        self.assertEqual(enums.first.value, 'first')
+        self.assertEqual(enums.second.value, 'second')
+        self.assertEqual(enums.third.value, 'third')
+        with self.assertRaisesRegexp(AttributeError, 'fourth'):
+            getattr(enums, 'fourth')
```

Comment:
Add a test that checks the `name` param too

Reply:
Will add that

Reply:
done